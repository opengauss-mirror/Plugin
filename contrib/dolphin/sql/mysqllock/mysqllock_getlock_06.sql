drop database if exists mysqllock;
create database mysqllock dbcompatibility 'b';
\c mysqllock
create extension dolphin;
--failed
select get_lock('slock',999999999999999999999999999999999999999999999.99999::numeric(50,5));
--failed
select get_lock('slock',-999999999999999999999999999999999999999999999.99999::numeric(50,5));
--failed
select get_lock('slock',-999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999);
--failed
select get_lock('slock',999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999);
select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::real);
select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::double precision);
select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::smallint);
select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::integer);
select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::decimal(10,5));
select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::numeric(10,5));
select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::real);
select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::double precision);
select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::smallint);
select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::integer);
select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::decimal(10,5));
select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::numeric(10,5));
\c postgres
drop database if exists mysqllock;
