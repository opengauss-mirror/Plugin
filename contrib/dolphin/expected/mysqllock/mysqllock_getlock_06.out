drop database if exists mysqllock;
NOTICE:  database "mysqllock" does not exist, skipping
create database mysqllock dbcompatibility 'b';
\c mysqllock
create extension dolphin;
--failed
select get_lock('slock',999999999999999999999999999999999999999999999.99999::numeric(50,5));
 get_lock 
----------
        1
(1 row)

--failed
select get_lock('slock',-999999999999999999999999999999999999999999999.99999::numeric(50,5));
 get_lock 
----------
        1
(1 row)

--failed
select get_lock('slock',-999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999);
 get_lock 
----------
        1
(1 row)

--failed
select get_lock('slock',999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::real);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::double precision);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::smallint);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::integer);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::decimal(10,5));
 get_lock 
----------
        1
(1 row)

select get_lock('slock',1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::numeric(10,5));
 get_lock 
----------
        1
(1 row)

select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::real);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::double precision);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::smallint);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::integer);
 get_lock 
----------
        1
(1 row)

select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::decimal(10,5));
 get_lock 
----------
        1
(1 row)

select get_lock('slock',-1.999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999::numeric(10,5));
 get_lock 
----------
        1
(1 row)

\c postgres
drop database if exists mysqllock;
